ansible/roles/install-microshift/defaults/main.yml

```diff
@@ -9,7 +9,7 @@ etcd_repo: https://github.com/openshift/etcd.git
 microshift_dir: "{{ ansible_env.HOME }}/microshift"
 microshift_repo: https://github.com/openshift/microshift.git
 microshift_rpms: []
-microshift_version: 4.13.0-rc-5
+microshift_version: 4.14.0-ec.4-202308011235.p0
```

Comment:
@sjug could you export this var to the `vars/all.yml` to be able to set any MicroShift version? thanks

Reply:
thanks, I'll change the MicroShift version by myself